fall23/research.md

```diff
@@ -0,0 +1,69 @@
+# Research Phase
+
+## Relevant Papers
+
+### **Face Aging with Conditional Generative Adversarial Networks (Antipov et al. 2017)**
+[Link to paper](https://arxiv.org/pdf/1702.01983.pdf)
+
+- **Summary:**
+    - The authors present a method for automatically aging faces while preserving the original person's identity (identity-preserving) using a Generative Adversarial Network (GAN) called Age Conditional Generative Adversarial Network (Age-cGAN).
+    - Age-cGAN, a GAN designed to generate high-quality synthetic images within specific age categories - is introduced in this paper.
+    - They propose a novel latent vector optimization approach that preserves the original person's identity during face aging.
+    - **Two-Step Face Aging Process:** 
+        1. Reconstructing an input face image using an optimal latent vector that preserves the original identity.
+        2. Generating the final aged face image by changing the age condition using the preserved identity.
+    - The paper explains how face reconstruction is performed using Age-cGAN and discusses the need for latent vector optimization.
+
+### **FaceShifter: Towards High Fidelity And Occlusion Aware Face Swapping (Li et al. 2020)**
+[Link to paper](https://arxiv.org/abs/1912.13457)
+
+- **Summary:**
+    - This paper introduces a novel approach for face swapping using a two-stage framework that aims to generate high-quality and realistic face swaps while addressing facial occlusions.
+    - **Methodology:**
+        1. In the first stage, the framework leverages the target image's attributes to synthesize a high-fidelity swapped face.
+        2. Further, to address the challenges of facial occlusions, the second stage introduces a Heuristic Error Acknowledging Refinement Network (HEAR-Net).
+    - The results demonstrate that the outcomes are not only visually appealing but also excel in preserving the subject's identity. Moreover, this approach is capable of generating realistic face images without subject-specific training, which will be very useful for our project.
+
+
+### **Face Aging with Identity-Preserved Conditional Generative Adversarial Networks**
+[Link to paper](https://ieeexplore.ieee.org/document/8578926)
+
+- **Summary:**
+    - The paper proposes an Identity-Preserved Conditional Generative Adversarial Networks (IPCGANs) framework for face aging, which aims to synthesize a face with a target age while preserving the identity of the input face.
+    - This paper tries to produce the face with specific age groups rather than a particular age and claims to be better than (Antipov et al. 2017) as mentioned above, we are aiming to check whether it is true and apply it to current pipeline to improve the face aging.
+
+
+
+## Data Requirements
+We request the following data:
+
+- Photos of currently missing children
```

Comment:
Can you mark which of these are available to you and which are not?

Reply:
Updated the file according to your suggestions :) @oindrillac